features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -275,7 +281,8 @@ Feature: Collection
   #   offers Tiny (Foo, Obscure), SGA (JS, TE)
   When I log out
     And I am logged in as "myname5"
-  When I go to the collections page
+  When all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.